src/test/java/uk/gov/digital/ho/proving/income/audit/statistics/PassStatisticsAccumulatorMultipleCallsTest.java

```diff
@@ -0,0 +1,220 @@
+package uk.gov.digital.ho.proving.income.audit.statistics;
+
+import org.junit.Test;
+import uk.gov.digital.ho.proving.income.audit.AuditResultByNino;
+import uk.gov.digital.ho.proving.income.audit.AuditResultType;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.List;
+
+import static com.google.common.collect.Iterables.concat;
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.util.Lists.newArrayList;
+import static uk.gov.digital.ho.proving.income.audit.AuditResultType.*;
+
+
+public class PassStatisticsAccumulatorMultipleCallsTest {
+
+    private static final LocalDate FROM_DATE = LocalDate.of(2019, Month.JANUARY, 1);
+    private static final LocalDate TO_DATE = LocalDate.of(2019, Month.JANUARY, 31);
+    private static final LocalDate IN_RANGE = FROM_DATE.plusDays(1);
+
+    private PassStatisticsAccumulator accumulator = new PassStatisticsAccumulator(FROM_DATE, TO_DATE);
+
+    @Test
+    public void accumulate_firstPassTooEarlySecondInRange_onlyCountOne() {
+        accumulator.accumulate(singleResult(FROM_DATE.minusDays(1), PASS));
+        accumulator.accumulate(singleResult(FROM_DATE, PASS));
+
+        assertThat(accumulator.result())
+            .isEqualTo(statisticsForCounts(1, 0, 0, 0));
+    }
+
+    @Test
+    public void accumulate_firstPassInRangeSecondTooLate_notCounted() {
```

Comment:
You'll have to explain the rules for me to understand this test

Reply:
The rules:

- An AuditResult is in the date range if its best result between the dates (inclusive). 
- The ranking of results is PASS > FAIL > NOTFOUND > ERROR
- Where we have multiple records with the same result, we take the most recent.

So in this case we have 2 PASS but as the latest is after the TO_DATE, we don't count the results for this Nino towards the stats.